jetpack-beta.php

```diff
@@ -982,15 +982,18 @@ static function run_autoupdate() {
 			if ( empty( $admin_email ) ) {
 				return;
 			}
-			$subject = 'Updated Jetpack Beta Tester';
+			$site_title = get_bloginfo( 'title' );
```

Comment:
Is this the proper way to get the title ?

Reply:
I think it is `get_bloginfo( 'name' )`

Reply:
but this one worked. The screenshot in the PR description is a real one. I'm gonna check.

Reply:
I think it should be `get_bloginfo( 'name' );`

Reply:
Its becasue the default value is name. :) so it the string you pass in doesn't mean anything then it just gives you back the name. 
See https://github.com/WordPress/WordPress/blob/master/wp-includes/general-template.php#L739

Reply:
hahah nice. I'v just updated